Classes/Views/UIView+Extensions.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UIView+Extensions.swift
+//  Freetime
+//
+//  Created by Joan Disho on 06.06.18.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+extension UIView {
+
+    func addSubviews(_ subviews: [UIView]) {
```

Comment:
I feel like this is overkill.

Reply:
And if we do want to keep it, some tests / header documentation would be nice.

Reply:
Why overkill? 🤔

I agree for the tests 👍

Reply:
Nit: but I don't like how the local variable name `subviews` shadows `self.subviews`

Reply:
Can we remove this? Unless this is fixing a real problem I don't understand.